src/openforms/js/components/form/edit/validationEditFormUtils.js

```diff
@@ -50,19 +50,6 @@ const getCustomValidationErrorMessagesEditForm = validators => {
     collapsible: true,
     collapsed: true,
     components: [
-      {
```

Comment:
😿

Reply:
tears were shed indeed, but not because it's been removed